demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -529,22 +539,40 @@ def _collect_layouts_dependencies(pack_layouts: list,
 
         for layout in pack_layouts:
             layout_data = next(iter(layout.values()))
+            layout_path = layout_data.get("file_path")
+            layout_info = get_json(layout_path)
```

Comment:
This might be bad when we run find dependencies on all content, it's another IO operation for each layout.
Don't we have any better way to identify the type?